pydantic/plugin/_schema_validator.py

```diff
@@ -17,8 +17,18 @@
 events: list[Event] = list(Event.__args__)  # type: ignore
 
 
+def build_type_path(module: str, name: str) -> str:
+    return f'{module}:{name}'
+
+
 def create_schema_validator(
-    schema: CoreSchema, config: CoreConfig | None = None, plugin_settings: dict[str, Any] | None = None
+    schema: CoreSchema,
+    source_type: Any,
+    module: str,
+    type_name: str,
+    item_type: str,
```

Comment:
rename and change type types here to. You can leave `schema_type_module` and `schema_type_name` as separate parameters if you like.